data/tabular/MUV_466/meta.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: MUV_466
```

Comment:
If you want you can put in the dataset names in this list to exclude them from adding the regression templates:
https://github.com/OpenBioML/chemnlp/blob/main/data/text_sampling/text_sampling.py#L70
(In the past I always did that, but if we do it right away I can't forget it. ;-) )
If we add it to this list only the text templates in the `meta.yaml` files are used for the text sampling.